<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NES Tool v 0.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="nestool.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QGroupBox" name="tileSetGroupBox">
      <property name="title">
       <string>Tileset</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="TileSet" name="tileSet" native="true">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>256</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="applyTilesCheckBox">
         <property name="text">
          <string>Apply tiles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="paletteGroupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Palettes</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="3" column="3">
        <widget class="QToolButton" name="paletteGButton">
         <property name="text">
          <string>G0</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QToolButton" name="paletteRButton">
         <property name="text">
          <string>R0</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2" colspan="5">
        <widget class="Palette" name="backgroundPalette" native="true">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="applyPalettesCheckBox">
         <property name="text">
          <string>Apply palettes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="paletteBButton">
         <property name="text">
          <string>B0</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="backgroundPaletteLabel">
         <property name="text">
          <string>Background:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QToolButton" name="paletteMButton">
         <property name="text">
          <string>M0</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="spritesPaletteLabel">
         <property name="text">
          <string>Sprites:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="Palette" name="spritesPalette" native="true">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Nametables</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QToolButton" name="moveNameTableLeftButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="addNameTableButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4">
        <widget class="QToolButton" name="moveNameTableRightButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QScrollArea" name="nameTableArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="nameTableContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>446</width>
            <height>486</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="NameTable" name="nameTable" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Remove_Duplicates"/>
    <addaction name="action_Remove_Unused"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Tileset">
    <property name="title">
     <string>&amp;Tileset</string>
    </property>
    <widget class="QMenu" name="menu_Recent_CHR">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
    </widget>
    <addaction name="action_Open_CHR"/>
    <addaction name="action_Save_CHR_As"/>
    <addaction name="action_Save_CHR"/>
    <addaction name="separator"/>
    <addaction name="menu_Recent_CHR"/>
   </widget>
   <widget class="QMenu" name="menu_Nametable">
    <property name="title">
     <string>&amp;Nametable</string>
    </property>
    <widget class="QMenu" name="menu_Recent_NameTable">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
    </widget>
    <addaction name="action_Open_NameTable"/>
    <addaction name="action_Save_NameTable"/>
    <addaction name="action_Save_All_NameTables"/>
    <addaction name="separator"/>
    <addaction name="menu_Recent_NameTable"/>
   </widget>
   <widget class="QMenu" name="menu_Palettes">
    <property name="title">
     <string>Pa&amp;lettes</string>
    </property>
    <widget class="QMenu" name="menu_Recent_Palettes">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
    </widget>
    <addaction name="action_Open_Palettes"/>
    <addaction name="action_Save_Palettes_As"/>
    <addaction name="action_Save_Palettes"/>
    <addaction name="separator"/>
    <addaction name="menu_Recent_Palettes"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Nametable"/>
   <addaction name="menu_Tileset"/>
   <addaction name="menu_Palettes"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_Palettes">
   <property name="text">
    <string>&amp;Open Palettes...</string>
   </property>
  </action>
  <action name="action_Save_Palettes">
   <property name="text">
    <string>&amp;Save Palettes</string>
   </property>
  </action>
  <action name="action_Open_CHR">
   <property name="text">
    <string>&amp;Open CHR...</string>
   </property>
  </action>
  <action name="action_Save_CHR_As">
   <property name="text">
    <string>&amp;Save CHR As...</string>
   </property>
  </action>
  <action name="action_Open_NameTable">
   <property name="text">
    <string>&amp;Open NameTable...</string>
   </property>
   <property name="toolTip">
    <string>Open NameTable</string>
   </property>
  </action>
  <action name="action_Save_NameTable">
   <property name="text">
    <string>&amp;Save NameTable As...</string>
   </property>
   <property name="toolTip">
    <string>Save NameTable</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="action_Remove_Duplicates">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove &amp;Duplicates</string>
   </property>
  </action>
  <action name="action_Remove_Unused">
   <property name="text">
    <string>Remove &amp;Unused</string>
   </property>
  </action>
  <action name="action_Save_All_NameTables">
   <property name="text">
    <string>Save &amp;All NameTables</string>
   </property>
  </action>
  <action name="action_Save_CHR">
   <property name="text">
    <string>Save &amp;CHR</string>
   </property>
  </action>
  <action name="action_Save_Palettes_As">
   <property name="text">
    <string>Save Palettes As...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TileSet</class>
   <extends>QWidget</extends>
   <header>tileset.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NameTable</class>
   <extends>QWidget</extends>
   <header>nametable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Palette</class>
   <extends>QWidget</extends>
   <header>palette.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="nestool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
